app/build.gradle

```
@@ -35,9 +35,9 @@ dependencies {
 
     implementation 'com.takisoft.preferencex:preferencex:1.1.0'
     implementation 'com.squareup.retrofit2:retrofit:2.9.0'
-    implementation "com.squareup.okhttp3:okhttp:4.10.0"
+    implementation 'com.squareup.okhttp3:okhttp:5.0.0-alpha.11'
```

I'd rather not use any alpha versions. Also unsure what effects updating okhttp major version could have on retrofit compatibillity

```suggestion
    implementation 'com.squareup.okhttp3:okhttp:4.10.0'
```